app/controllers/api/v1/books_controller.rb

```diff
@@ -11,6 +13,11 @@ def show
         render json: @book
       end
 
+      def isbn
+        open = OpenLibraryService.new
+        render json: open.book_info(params[:isbn])
```

Comment:
Use strong params here `params.permit(:isbn)` and define it inside another params method